app/views/RemoveIndexView.scala.html

```diff
@@ -17,29 +17,32 @@
 @import controllers.routes._
 @import models.Mode
 @import play.api.mvc.Call
+@import views.html.components.{ErrorSummary, InputYesNo, SubmitButton}
 
 @this(
     main_template: MainTemplate,
-    formHelper: FormWithCSRF
+    formHelper: FormWithCSRF,
+    errorSummary: ErrorSummary,
+    inputYesNo: InputYesNo,
+    submitButton: SubmitButton
 )
 
 @(prefix : String, form: Form[_], index: Int, nameOrDescription : String, route : Call)(implicit request: Request[_], messages: Messages)
 
 @main_template(
-    title = s"${errorPrefix(form)}${messages(s"$prefix.title", nameOrDescription)}"
-    ) {
+    title = s"${errorPrefix(form)}${messages(s"$prefix.title", nameOrDescription)}",
```

Comment:
Check the `prefix` in the controller - I think you'll need to add YesNo to it, and the message keys so that the error summary link works

Luckily since it's RemoveIndexView you won't need to also make changes in other repos as outlined here: https://github.com/hmrc/register-trust-other-individual-frontend/blob/master/Migration%20to%20GovUk%20Frontend.md#draft-and-confirm-print